applications/FluidDynamicsApplication/custom_strategies/strategies/residualbased_predictorcorrector_velocity_bossak_scheme_turbulent.h

```diff
@@ -115,24 +186,13 @@ namespace Kratos {
             double MoveMeshStrategy,
             unsigned int DomainSize)
         :
-          Scheme<TSparseSpace, TDenseSpace>(),
+          ResidualBasedBossakVelocityScheme<TSparseSpace, TDenseSpace>(NewAlphaBossak),
           mRotationTool(DomainSize,DomainSize+1,IS_STRUCTURE,0.0), // Second argument is number of matrix rows per node: monolithic elements have velocity and pressure dofs.
           mrPeriodicIdVar(Kratos::Variable<int>::StaticObject())
```

Comment:
I would argue that periodicity should be handled by MPCs

Reply:
@jcotela ? It was there when I ported it to the generic scheme.

Reply:
yes...consider this comment as a reminder

Reply:
I would leave it on the derived fluid scheme for now.